test/meson.build

```diff
@@ -25,7 +25,3 @@ if gtest_dep.found() and not meson.is_cross_build()
              workdir: meson.current_build_dir())
     endforeach
 endif
-
-if get_option('default_library') != 'static' and not meson.is_cross_build()
-    subdir('pytest')
-endif
```

Comment:
I would prefer to have the python tests removed one by one in the same commit that add the cpp tests testing the same feature. It would allow us to better identify the equivalence of the tests.

Reply:
Done